docroot/sites/admissions.uiowa.edu/modules/admissions_core/sass/pdf.scss

```diff
@@ -478,6 +473,15 @@ h6 {
   }
 }
 .node--type-page {
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
+    font-family: $font-family-serif;
```

Comment:
Do you mind removing the uppercase on these?  I don't think we have any styles for Zilla that use all caps.